new-dti-website-redesign/src/app/test-components/page.tsx

```diff
@@ -253,6 +254,78 @@ export default function TestComponents() {
           </IconWrapper>
         </div>
       </div>
+
+      <div className="flex flex-col gap-6">
+        <h4>Feature Card</h4>
+        <div className="flex gap-0">
```

Comment:
Nit: by default, `flex` doesn't add any gap between child elements, so we don't need to to explicitly have `gap-0`
```suggestion
        <div className="flex">
```